server/devdocs/bin/generate-proptypes-index.js

```diff
@@ -30,6 +30,16 @@ const camelCaseToSlug = ( name ) => {
 		.replace( /^-/, '' );
 };
 
+const slugToCamelCase = name => {
+	if ( ! name ) {
+		return name;
+	}
+
+	return name
```

Comment:
Ahh. I wonder if we could reuse that module, did it actually work when you tried to import it?